doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1158,20 +1158,29 @@ Response code:
 A subscription is represented by a JSON object with the following fields:
 
 * `id`: Subscription unique identifier. Automatically created at creation time.
-* `subject` : It s an object that describes the subject of the subscription.
-* `notification` : It is an object that describes the notification received by the subscriber.
+* `subject`: It is an object that describes the subject of the subscription.
+* `notification`: It is an object that describes the notification received by the subscriber.
 * `duration`: Duration of the subscription in ISO8601 format. Infinite if not specified.
 
-A `subject` contains the following subfields:
+The `subject` support two variants, to subscribe a single entity id/pattern or to subscribe a list of them.
+A `subject` for subscribing a single id/pattern has the following subfields (`id`/`idPattern` or `type` must be present):
 
 * `id` or `idPattern` : Id or pattern of the affected entities (optional). Both cannot be used at the same time.
 * `type` : Type of the affected entities (optional).
 * `condition`: Condition that will trigger the notification. It can have two optional properties:
  * `attributes`: array of attribute names
  * `expression`: an expression composed of `q`, `geometry` and `coords` (see "List entities" operation above
-about this field):
+about this field).
 
-`id`/`idPattern` or `type` must be present.
+A `subject` for subscribing a list of ids/patterns has the following subfields (`entities` must be present and not empty):
```

Comment:
After skype discussion, we will use only `entities`. The old case for only one id/idPattern will be covered with the `entities` vector having just 1 element.